mods/ra/maps/allies-03a/rules.yaml

```diff
@@ -175,10 +175,6 @@ KENN:
 	Buildable:
 		Prerequisites: ~disabled
 
-E3:
-	Buildable:
-		Prerequisites: ~disabled
```

Comment:
Removing this enables them for this mission. Is that wanted?

Reply:
It's moved to ~tent, which are unavailable in this mission.